test/tools/libtesteth/TestHelper.cpp

```diff
@@ -434,7 +476,7 @@ void executeTests(const string& _name, fs::path const& _testPathAppendix, fs::pa
 		json_spirit::read_string(s, v);
 		removeComments(v);
 		json_spirit::mValue output = doTests(v, true);
-		addClientInfo(output, testfilename);
+		addClientInfo(output, testfileUnderTestPath);
```

Comment:
Looks like I forgot this change in a merge conflict.  Thanks.

Reply:
thats why conflicts on rebase PR are bad